src/Foundatio.Repositories/Models/Aggregations/IAggregate.cs

```diff
@@ -5,6 +5,7 @@
 namespace Foundatio.Repositories.Models;
 
 [JsonConverter(typeof(AggregationsJsonConverter))]
+[System.Text.Json.Serialization.JsonConverter(typeof(AggregationsSystemTextJsonConverter))]
```

Comment:
Maybe we should alias the 2 different `JsonConverter` types like maybe `NSJ.JsonConverter` and `STJ.JsonConverter`?

Reply:
Or maybe just fully qualify both of them.